### ISSUE TITLE
[4.5.3]: Default radio button option not applied when field is added to an existing entry

### ISSUE THREAD

**Issue by thupsi** [craftcms/cms#13689]
### Description

When adding a `radio buttons` field with a default option, this option is not applied to existing entries.

### Steps to reproduce

1. Create a custom field with type `radio buttons`, at least 2 options, one of wich is set as `default`.
2. Add this field to a section's field layout.
3. Edit an existing entry in this section.

### Expected behavior

The default option should be selected

### Actual behavior

No option is selected

### Craft CMS version

4.5.3

### Installed plugins and versions

-

**Comment by brandonkelly** [comment]
This is currently working as intended, but it’s worth us reconsidering, in the same vein as the Dropdown behavior change in 4.5. We’ll discuss internally…

**Comment by thupsi** [comment]
Yes, I was indeed comparing it to the current dropdown behaviour. I'm sure you have a much clearer picture than me about possible drawbacks and implications, so I trust you will handle it in the best possible way.
